apps/dashboard/test/system/jobs_app_test.rb

```diff
@@ -76,22 +76,22 @@ def setup
       find('#product_icon_select').set(icon)
       click_on 'Save'
 
-      assert_selector '.alert-danger', text: 'Project name may only contain letters, digits, dashes, and underscores'
+      assert_selector '.alert-danger:not(.close)', text: 'Project name may only contain letters, digits, dashes, underscores, and spaces'
     end
   end
 
   test 'create with invalid name trigers alert' do
     Dir.mktmpdir do |dir|
       OodAppkit.stubs(:dataroot).returns(Pathname.new(dir))
-      proj = 'test project'
+      proj = 'test & project'
```

Comment:
Come to think of it, I think ampersands should be OK too. Which begs the question, what _shouldn't_ be allowed? Anything?  I get the feeling most things should be OK save for maybe newlines?